docs/machine-learning/resources/tasks.md

```diff
@@ -0,0 +1,46 @@
+# Machine Learning Tasks
+After understanding what you are hoping to achieve with your data, you will need to pick the right machine learning task for your situation. Below, you can find an explanation of different machine learning tasks and some common use cases. 
+
+> [!NOTE]
+> ML.NET is currently in Preview. All machine learning tasks are not currently supported. To submit a request for a certain task, please open an issue on our [GitHub repository](https://github.com/dotnet/machinelearning/issues).
+
+## Binary Classification
+A supervised machine learning task that is used to predict which of two classes an instance of data belongs to. The input of a classification algorithm is a set of labeled examples. Each example is represented as a feature vector, and each label is an integer of value of 0 or 1. The output of a binary classification algorithm is a classifier, which can be used to predict the label of new unlabeled instances. Examples of binary classification scenarios include:
+
+* Understanding sentiment of Twitter comments as either "positive" or "negative"
+* Diagnosing whether a medical patient has a certain disease or not
+* Making a decision to send an email to a "spam" folder or not
+
+## Multi-class Classification
+A supervised machine learning task that is used to predict the category of an instance of data. The input of a classification algorithm is a set of labeled examples. Each example is represented as a feature vector, and each label is an integer between 0 and k-1, where k is the number of classes. The output of a classification algorithm is a classifier, which can be used to predict the label of a new unlabeled instance. Examples of multi-class classification scenarios include:
+
+* Determining the breed of a dog as a "Siberian Husky", "Golden Retriever", "Poodle", etc.
+* Understanding movie reviews as "positive", "neutral", or "negative"
+* Categorizing hotel reviews as "location", "price", "cleanliness", etc.
+
+> [!NOTE]
+> Currently, ML.NET does not support classification tasks for images. Support will be added in a future release. 
```

Comment:
*does not* -> *doesn't* 
(also in other notes)

nit: *in a future release* -> *in future releases* to clearly state that it might happen not in the next release; I can imagine that *a future release* might be misunderstood as *the next release*